app/javascript/packages/document-capture/components/review-issues-step.jsx

```diff
@@ -181,7 +187,9 @@ function ReviewIssuesStep({
       }
     >
       {!!unknownFieldErrors &&
-        unknownFieldErrors.map(({ error }) => <p key={error.message}>{error.message}</p>)}
+        unknownFieldErrors
+          .filter((error) => !['front', 'back', 'selfie'].includes(error.field))
```

Comment:
@aduth This is what I was talking about. Because there are no fields registered at this point on the warning error screen all the errors are considered "unknownFieldsErrors" and thus without this filter it shows all the messages. This is inelegant but gets the job done though there must be a better way.

Reply:
> @aduth This is what I was talking about. Because there are no fields registered at this point on the warning error screen all the errors are considered "unknownFieldsErrors" and thus without this filter it shows all the messages. This is inelegant but gets the job done though there must be a better way.

Hm, yeah, I see the challenge. I think this _could_ be okay, since it would be expected that this component should be aware of the specific fields it's prepared to handle. The part that would have me worried is that the `FormSteps` would still consider these as "unknown", when in reality they're not. For example, that might be making it so that the [default behavior of "don't allow submit until field errors are resolved"](https://github.com/18F/identity-idp/blob/31c533ffc12e48d20087de0f8cc6a90445962df3/app/javascript/packages/document-capture/components/form-steps.jsx#L223-L228) isn't being enforced correctly.

One alternative could be to slightly adjust how we do field registration from being part of the render (i.e. above, `ref={registerField(...)}`) instead as a hook, which is always invoked regardless whether the field is being rendered at the time.

```jsx
const selfieRef = useField('selfie', { isRequired: true });

// ...

return  <AcuantCapture ref={selfieRef} />;
```

That way, at least `FormSteps` can know about the fields in a step, even if they're not visible at the time.